lib/ui/painting/fragment_shader.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
The other file you added has a copyright of 2013, this should be consistent with whatever existing files have in here I think.